Swift Chat Demo 2.xcodeproj/project.pbxproj

```diff
@@ -458,12 +470,17 @@
 			baseConfigurationReference = 26D03EB7DD24CC71FAF6D1C6 /* Pods-Swift Chat Demo 2.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				CODE_SIGN_ENTITLEMENTS = "Swift Chat Demo 2/Swift Chat Demo 2.entitlements";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
+				CODE_SIGN_STYLE = Manual;
```

Comment:
After we test it and it works. Please revert the provisioning selection to Automatic, as developers may have their own cert. So we don't have to change it when updating the cert and developers can also use their own certs.